src/views/News/index.jsx

```diff
@@ -317,12 +337,37 @@ const StudentNews = () => {
       content = <GordonLoader />;
     } else {
       content = (
-        <NewsList
-          news={news}
-          personalUnapprovedNews={personalUnapprovedNews}
-          handleNewsItemEdit={handleNewsItemEdit}
-          handleNewsItemDelete={handleNewsItemDelete}
-        />
+        <>
+          {isAdmin && (
+            <NewsList
+              news={unapprovedNews}
+              header={'All Pending Posts'}
```

Comment:
```suggestion
              header={'All Pending News'}
```
The other two blocks use the word `News`. I think it would be good to keep that consistent.